config/cloudwatch-exporter/config.yml

```diff
@@ -0,0 +1,35 @@
+---
+discovery:
+  exportedTagsOnMetrics:
+    ec2:
+      - Name
+    rds:
+      - Name
+  jobs:
+    - type: "ec2"
+      region: $AWS_REGION
```

Comment:
I think it would be clearer that this is replaced with `sed` if we used a different delimiter (ie not a `$`).  Maybe something like `__AWS_REGION__`.

Reply:
We'll use spruce as it's available in the Docker image.